src/extractMeta.js

```diff
@@ -7,9 +8,12 @@ const extractMetadata = props => {
             return;
         }
         const propType = props[key].type ? props[key].type : props[key];
+        const label = props[key].label === undefined
+            ? fromCamelToSentence(key)
+            : props[key].label;
```

Comment:
Would this return a string of `'false'` for `label: false`?

Reply:
If `props[key].label` is `false`, `""`, or `null` it passes through `false`, `""`, or `null`. There's a unit test that verifies that.

Reply:
The other side of this is the FormFromProps method that decides if we should show a label. Which is why it's best we handle the backwards compatibility here, because what we're seeing on Oransi is what happens when Site Designer sees a falsy value.
```public isLabelVisible = (label: IndividualPropLabel): boolean => {
    return Boolean(label);
};```

Reply:
Ok, so if the label is undefined, then it will generate the label based on the key. It will then feed the label (whatever it may be) into Site Designer and check for any other falsy values to see if the label should be visible or not. That's correct?

Reply:
Correct. Site Designer hides any labels with a falsy value, as we saw with those older blocks on Oransi where the labels are undefined. Since being able to hide a label by setting it to `false` or `""` is a feature that we want, it's important we handle `undefined` here as a special legacy case. (I'm just re-stating what you've already said.)